src/quic/node_quic_crypto.cc

```diff
@@ -650,17 +651,24 @@ void SetHostname(SSL* ssl, const std::string& hostname) {
 }  // namespace
 
 void InitializeTLS(QuicSession* session) {
+  static bool warn_trace_tls = true;
```

Comment:
This should not be global mutable state, and stored on the `Environment` like we do for other warnings we only want to emit once